docs/content/v0.20/release.md

```diff
@@ -1,6 +1,6 @@
 # Making KubeStellar Releases
 
-This document defines how releases of the KubeStellar repository are made.
+This document defines how releases of the KubeStellar repository are made. This document is a work-in-progress. In particular, the dependency cycle between the `kubestellar` and `ocm-tansport-plugin` repos is not well documented and we have not agreed on how to test and release in the presence of this cycle.
```

Comment:
nit: Maybe replace the "we have not agreed"  ? 
Maybe " ....repos is not well documented and  it's still not clear how to test and release in the presence of this cycle. "

Reply:
done